includes/cli/class-amp-cli-validation-command.php

```diff
@@ -267,6 +194,129 @@ public function run( $args, $assoc_args ) {
 		WP_CLI::line( sprintf( 'For more details, please see: %s', $url_more_details ) );
 	}
 
+	/**
+	 * Provides the associative args for the command.
+	 *
+	 * @return array
+	 */
+	private function get_assoc_args() {
+		if ( ! is_array( $this->assoc_args ) ) {
+			$this->assoc_args = [];
+		}
+
+		return array_merge(
+			[
+				self::LIMIT_URLS_ARGUMENT        => 100,
+				self::INCLUDE_ARGUMENT           => [],
+				self::FLAG_NAME_FORCE_VALIDATION => false,
+			],
+			$this->assoc_args
+		);
+	}
+
+	/**
+	 * Provides the ValidationURLProvider instance.
+	 *
+	 * @return ValidationURLProvider
+	 */
+	private function get_validation_url_provider() {
+		if ( ! is_null( $this->validation_url_provider ) ) {
+			return $this->validation_url_provider;
+		}
+
+		$assoc_args = $this->get_assoc_args();
+
+		$include_conditionals      = [];
+		$force_crawl_urls          = false;
```

Comment:
This can immediately be set to the force flag and adapted as needed later.
```suggestion
		$force_crawl_urls          = Utils::get_flag_value( $assoc_args, self::FLAG_NAME_FORCE_VALIDATION, false );
```